sklearn/decomposition/_kernel_pca.py

```diff
@@ -56,10 +63,32 @@ class KernelPCA(TransformerMixin, BaseEstimator):
         Learn the inverse transform for non-precomputed kernels.
         (i.e. learn to find the pre-image of a point)
 
-    eigen_solver : string ['auto'|'dense'|'arpack'], default='auto'
-        Select eigensolver to use. If n_components is much less than
-        the number of training samples, arpack may be more efficient
-        than the dense eigensolver.
+    eigen_solver : string {'auto', 'dense', 'arpack', 'randomized'}, \
+        default='auto'
```

Comment:
is this the correct way to do a multiline parameter description header ?

Reply:
yes